zha/units.py

```diff
@@ -157,3 +157,6 @@ class UnitOfEnergy(StrEnum):
 
 # Percentage units
 PERCENTAGE: Final[str] = "%"
+
+# Electrical conductivity units
+ELECTRICAL_CONDUCTIVITY_DECISIEMENS_PER_METER: Final[str] = "dS/m"
```

Comment:
We tend to closely follow what Home Assistant does here. We should use `UnitOfConductivity`. See HA:
https://github.com/home-assistant/core/blob/d657809ffedd61d02b8c17a480daca9894ab4ce3/homeassistant/const.py#L809-L822 (ignore the deprecated part though)

It's already in zigpy (for quirks v2*):
https://github.com/zigpy/zigpy/blob/eb6ba8079d1e66d882efc132b3d0859d12e20c7d/zigpy/quirks/v2/homeassistant/__init__.py#L199-L205
We should copy that into here for now.

(*: Yes, this needs to be consolidated at one point. ZHA should likely just use this from zigpy, so it's not duplicated. But that's something for another PR. We should also make sure to update everything from HA then.)

Reply:
Done.